content/blog/zephyrs-summer-intern-experience-with-pulumi/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Zephyr's Summer Intern Experience with Pulumi"
+date: "2020-09-17"
+meta_desc: "Zephyr's internship experience in Summer 2020, personal growth, skills learned and reflection"
+meta_image: zephyr-pulumi.png
+authors: ["zephyr-zhou"]
+tags: ["pulumi","intern"]
+---
+
+Hi, I am [Zephyr Zhou](https://www.linkedin.com/in/zephyr-zhou-a17741196/), a senior Computer Science student at the University of Washington. I have just spent this entire Summer interned at Pulumi. This is my first internship ever in my life. Thanks to Pulumi for providing this opportunity even in this difficult time of the Covid-19 epidemic. Despite the sad truth that I couldn't get in touch offline, I believe this would be one of the most precious memory.
+
+Time goes by so fast, but before saying goodbye to my internship, I would like to share the story.
+
+<!--more-->
+
+## Personal Growth
+
+Honestly, before my internship start, I was so nervous that I couldn't sleep. Previously, I had taken only one data management course that contains a tutorial about set up Azure SQL Database. Other than that, I have known nothing about the cloud. Although I have read about the [Getting Started documentation](https://www.pulumi.com/docs/get-started/aws/) and Go, I was worried that my understanding was not good enough. Also, I transferred from a community college to the University of Washington as a junior student and was admitted to the Computer Science major after that. Therefore, I felt relatively new to programming compared to my peers and less confident working in the industry, even less working remotely.
+
+However, I am glad that I was brave enough to make the first step out of my comfort zone to learn about new things and improve my understanding during practicing. That was one of the important reasons for me to choose to intern at Pulumi, a startup with great potential. Also, due to the nature of startups, Pulumi has smaller but more connected teams. Personally, I felt more comfortable discussing issues and thoughts in small groups. Also, I was able to work with the different teams and people. I was also able to broaden my horizons and improve my collaboration skills.
+
+I found that the more familiar I got with the company and the job, the more confident I was. This is also indispensable from the support of company members. The warm greetings and kind help gave me a lot of powers and courage. Working remotely with Pulumi was much more interactive than I thought. The one-to-one meeting with my mentor and site supervisor has boosted my working efficiency because I could compile my questions and updates, talk about them all together, and get informed about if I am on the right path. I also loved the fact that I was able to join other team meetings so I could learn about the different projects other people are working on.
```

Comment:
```suggestion
I found that the more familiar I got with the company and the job, the more confident I was. This is also indispensable from the support of company members. The warm greetings and kind help gave me a lot of power and courage. Working remotely with Pulumi was much more interactive than I thought. The one-to-one meeting with my mentor and manager has boosted my working efficiency because I could compile my questions and updates, talk about them all together, and get informed about if I am on the right path. I also loved the fact that I was able to join other team meetings so I could learn about the different projects other people are working on.
```

Reply:
I'm not quite sure what you meant to say with "This is also indispensable from the support of company members." -- Could this be phrased as "My confidence was boosted in large part from the support of team members."?